api/v1beta1/verticadb_webhook.go

```diff
@@ -1011,6 +1038,18 @@ func (v *VerticaDB) checkImmutableS3ServerSideEncryption(oldObj *VerticaDB, allE
 	return allErrs
 }
 
+// checkImmutableDepotVolume will make sure local.depotVolume
+// does not change after creation.
+func (v *VerticaDB) checkImmutableDepotVolume(oldObj *VerticaDB, allErrs field.ErrorList) field.ErrorList {
+	if v.Spec.Local.DepotVolume != oldObj.Spec.Local.DepotVolume {
+		err := field.Invalid(field.NewPath("spec").Child("local").Child("depotVolume"),
+			v.Spec.Local.DepotVolume,
+			"local.depotVolume cannot change after creation")
```

Comment:
Similar comment here about repeating the field name.

Here is the output if you try to violate this rule:
```
# verticadbs.vertica.com "v" was not valid:
# * spec.local.depotVolume: Invalid value: "PersistentVolume": local.depotVolume cannot change after creation
```

Suggest the following:
```suggestion
			"cannot change after creation")
```